### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by Hnh013**
added components : Navigation bar , Category , Footer , Banner
added Page : Home
added routing for home page

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
good code structure and variable naming conventions,
code looks modular
can be merged

File: src/frontend/styles/components/banner.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+.banner-container {
+    overflow-x: hidden;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 350px;
```
Comment (tejas-muthya37) [inline src/frontend/styles/components/banner.css]: Can use rem here.

**Review by Shah-Arsalan — APPROVED** [review]
Code seems to be clean . Nothing that seems out of place . You can merge

**Review by tejas-muthya37 — APPROVED** [review]
Did not any issues with the code. Looks good to merge.

File: src/frontend/styles/components/banner.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  
+  .banner-overlay-div {
+    display: grid;
+    max-width: 100%;
+    background: black;
+    background: linear-gradient(
+      60deg,
+      rgba(0, 0, 0, 0.7777485994397759) 30%,
+      rgba(255, 255, 255, 0) 100%
+    );
```
Comment (OmKakatkar) [inline src/frontend/styles/components/banner.css:33]: Try not to use color names directly, use css variables instaed for consistency.

**Review by OmKakatkar — APPROVED** [review]
Nice to see barrel files
Good naming conventions
Like the folder structure

Looks good to merge

File: src/frontend/styles/components/banner.css (hunk 1 above)
Comment (Hnh013) [reply src/frontend/styles/components/banner.css]: sure tejas will keep this in mind !

File: src/frontend/styles/components/banner.css (hunk 2 above)
Comment (Hnh013) [reply src/frontend/styles/components/banner.css:33]: thanks Om, will change this to a variable i, didn't did it initally as this is the one and only place its being used .

**Review by Aman-Jain1996 — COMMENTED** [review]
clean code with good folder structure 
 suggestion - try to use one unit for font-sizes

**Pull request merged**